### PR TITLE
lambda-sns-sls example

### PR DESCRIPTION
**Pull request by extremenelson**
*Issue #, if available:*

*Description of changes:*
Added in example of using the Serverless Framework to create a SNS topic and a lambda to send messages to the topic.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by extremenelson** [comment]
Added in example of using the Serverless Framework to create a SNS topic and a lambda to send messages to the topic.

**Comment by boyney123** [comment]
Hi @extremenelson ,

Thanks for the contribution.

Can you add the `example-pattern.json` file to your two patterns? You can find more information in our contribution guidelines https://github.com/aws-samples/serverless-patterns/blob/main/.github/ISSUE_TEMPLATE/new-serverless-pattern-submission.md

image

Thanks, after that is done I will review, and test the patterns and get them merged 👍

**Comment by extremenelson** [comment]
Added in example-pattern.json

**Comment by extremenelson** [comment]
Added in example for Lambda putting events to EventBridge using the Serverless Framework

**Comment by boyney123** [comment]
Thanks looking good.

Last thing, could you also add an `example.json` file to the `lambda-sns` pattern?

**Comment by extremenelson** [comment]
The example-patterns are now added.

File: lambda-eventbridge-sls/README.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+1. Create a new directory, navigate to that directory in a terminal and clone the GitHub repository:
+   ```
+   git clone https://github.com/aws-samples/serverless-patterns
+   ```
+1. Change directory to the pattern directory:
+   ```
+   cd lambda-eventbridge-sls
+   ```
+1. Using the editor of your choice, oopen up the serverless.yml file and change any of the configuration settings to match your environment.
```
Comment (boyney123) [inline lambda-eventbridge-sls/README.md:28]:
```suggestion
1. Using the editor of your choice, open up the serverless.yml file and change any of the configuration settings to match your environment.
```

File: lambda-eventbridge-sls/example-pattern.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  "introBox": {
+    "headline": "Create a Lambda function that sends events to EventBridge.",
+    "text": [
+      "The Serverless template deploys a Lambda function with permission to publish events to EventBridge. The Lambda function publishes an event to the default bus in EventBridge when invoked."
+    ]
+  },
+  "gitHub": {
+    "template": {
+      "repoURL": "https://github.com/aws-samples/serverless-patterns/tree/main/lambda-eventbridge-sls",
+      "templateURL": "serverless-patterns/lambda-eventbridge-sls"
```
Comment (boyney123) [inline lambda-eventbridge-sls/example-pattern.json:16]:
```suggestion
      "templateURL": "serverless-patterns/lambda-eventbridge-sls",
      "projectFolder": "lambda-eventbridge-sls",
      "templateFile": "serverless.yml"
```

Comment (boyney123) [reply lambda-eventbridge-sls/example-pattern.json:16]: Missing some fields for the file

File: lambda-sns-sls/example-pattern.json [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  "introBox": {
+    "headline": "Create a Lambda function that sends events to a SNS topic.",
+    "text": [
+      "The Serverless template deploys a SNS topic and a Lambda function that has permission to publish events to the topic."
+    ]
+  },
+  "gitHub": {
+    "template": {
+      "repoURL": "https://github.com/aws-samples/serverless-patterns/tree/main/lambda-sns-sls",
+      "templateURL": "serverless-patterns/lambda-sns-sls"
```
Comment (boyney123) [inline lambda-sns-sls/example-pattern.json:16]:
```suggestion
      "templateURL": "serverless-patterns/lambda-sns-sls",
       "projectFolder": "lambda-sns-sls",
       "templateFile": "serverless.yml"
```

File: lambda-sqs-sls/example-pattern.json [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+      "Confirm the stack has been deleted: aws cloudformation list-stacks --query \"StackSummaries[?contains(StackName,'STACK_NAME')].StackStatus\""
+    ]
+  },
+  "authors": [
+    {
+      "name": "Alex Nelson",
+      "bio": "Senior Solutions Architect, AWS"
+    }
+  ]
+}
```
Comment (boyney123) [inline lambda-sqs-sls/example-pattern.json:49]: Think this pattern is already live https://serverlessland.com/patterns/lambda-sqs-python-sls

We can remove this folder.

File: lambda-ssm-sls/README.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
+# AWS Lambda GET and PUT to Amazon SSM Parameter Store Using the Serverless Framework
```
Comment (boyney123) [inline lambda-ssm-sls/README.md:1]: Missing example-pattern.json on this pattern, if you could add that would be great :)

File: lambda-ssm-sls/README.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+1. Invoke the Lambda function to get the current value from the SSM Parameter Store:
+
+```bash
+curl http://YOUR_LAMBDA_GET_ENDPOINT
+```
+
+2. Update the value in the SSM Parameter Store
+
+```bash
+curl -X POST http://YOUR_LAMBDA_PUT_ENDPOINT -x "YOUR_NEW_VALUE"
```
Comment (boyney123) [inline lambda-ssm-sls/README.md:48]:
```suggestion
curl -X PUT http://YOUR_LAMBDA_PUT_ENDPOINT -d "YOUR_NEW_VALUE"
```

**Pull request merged**